packages/neutron-sdk/src/bindings/oracle/types.rs

```diff
@@ -0,0 +1,23 @@
+use cosmwasm_std::Int128;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct CurrencyPair {
```

Comment:
The same type defined in `marketmap` module. Does it make sence to extract it in a common package?